blueprints/validator/files/tests/unit/validators/__name__-test.js

```diff
@@ -1,9 +1,8 @@
 import Ember from 'ember';
-import { moduleFor, test }
-from 'ember-qunit';
+import { moduleFor, test } from 'ember-qunit';
 
 moduleFor('validator:<%= dasherizedModuleName %>', 'Unit | Validator | <%= dasherizedModuleName %>', {
-  needs: ['validator:messages']
+  integration: true
```

Comment:
Gotcha yeah I agree. Let me change the folder name to `integration` for now. In the future if want to change it back to unit, then we can make the switch back.